src/app/services/ci-dashboard.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from "@angular/core";
+import { HttpClient } from "@angular/common/http";
+import { map } from "rxjs/operators";
+import { Meta,Title } from '@angular/platform-browser';
+
+@Injectable()
+export class DashboardData {
+
+    private apiurl: string;
+    constructor(private http: HttpClient, private meta:Meta,private titleService: Title) {
+        this.apiurl = "http://147.75.102.37:30002";
```

Comment:
This api url will not work for openebs.ci, this is only for test purpose, we have to deploy the backend in the workload cluster